### ISSUE TITLE
Make all fails due to missing generator/wg_index.tmpl

### ISSUE THREAD

**Issue by mwielgus** [kubernetes/community#754]
```
$ make all
[....]
Step 1/1 : RUN go-wrapper install
 ---> Running in 3ceb14ec0693
+ exec go install -v
gopkg.in/yaml.v2
app
 ---> a836b1f8c4df
Removing intermediate container 6afb7f1cc1cb
Removing intermediate container 7919eb4988e4
Removing intermediate container 3ceb14ec0693
Successfully built a836b1f8c4df
docker run -v /Users/mwielgus/src/community:/go/src/app sigdocs
+ exec app
Generated sig-api-machinery/README.md
Generated sig-aws/README.md
Generated sig-apps/README.md
Generated sig-auth/README.md
Generated sig-autoscaling/README.md
Generated sig-azure/README.md
Generated sig-big-data/README.md
Generated sig-cli/README.md
Generated sig-cluster-lifecycle/README.md
Generated sig-cluster-ops/README.md
Generated sig-contributor-experience/README.md
Generated sig-docs/README.md
Generated sig-federation/README.md
Generated sig-instrumentation/README.md
Generated sig-network/README.md
Generated sig-node/README.md
Generated sig-on-premise/README.md
Generated sig-openstack/README.md
Generated sig-product-management/README.md
Generated sig-scalability/README.md
Generated sig-scheduling/README.md
Generated sig-service-catalog/README.md
Generated sig-storage/README.md
Generated sig-testing/README.md
Generated sig-ui/README.md
Generated sig-windows/README.md
2017/06/22 00:48:26 open generator/wg_index.tmpl: no such file or directory
make: *** [gen-docs] Error 1
bash-3.2$ git diff
```

**Comment by cblecker** [comment]
Looks like https://github.com/kubernetes/community/pull/680 may have broke things

@vishh

**Comment by vishh** [comment]
Posted a patch - https://github.com/kubernetes/community/pull/755
Sorry about the issue!

**Comment by jamiehannaford** [comment]
This is also fixed by #718

**Comment by cblecker** [comment]
Thanks to you both. Will review these this morning!
